blobstore/list.go

```diff
@@ -113,7 +123,7 @@ func (bh *BlobHandler) HandleListByPrefix(c echo.Context) error {
 	}
 
 	log.Info("Successfully retrieved list by prefix:", prefix)
-	return c.JSON(http.StatusOK, objectKeys)
+	return c.JSON(http.StatusOK, result)
 }
 
 // HandleListByPrefixWithDetail retrieves a detailed list of objects in the specified S3 bucket with the given prefix.
```

Comment:
Why does this function not also take the delimiter as an option? I would think this would be the exact same function except that you return more information?

Reply:
The HandleListByPrefixWithDetail function is tailored for the UI and always sets the delimiter to true by default. However, I've now added the option to set the delimiter manually. If you don't specify a value, it will default to true. Thank you!